src/Lessons-api.Data/Repositories/StudentRepository.cs

```diff
@@ -19,14 +20,14 @@ public StudentRepository(LessonsContext lessonsContext)
 
         public async Task<StudentEntity> GetStudentById(int id)
         {
-            var result = await _lessonsContext.Students.FindAsync(id);
+            var result = await _lessonsContext.Students.Where(s => s.Id == id).Include(s => s.User).FirstOrDefaultAsync();
```

Comment:
Use `FindAsync()`

Reply:
When I use FindAsync () I have some fields null, I did this for include ().

Reply:
You can can use `FindAsync` with include